app/components/topicPicker.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import {RouteHandler} from 'react-router';
 import objectAssign from 'object-assign';
 import ListenerMixin from 'alt/mixins/ListenerMixin';
+import jquery from 'jquery';
```

Comment:
actually, eslint complains since `$ is already declared in the upper scope`.

someone else is declaring `$`. I'll just stick to `var $ = jquery` later in the code.